spython/version.py

```diff
@@ -18,3 +18,9 @@
 INSTALL_REQUIRES = (
     ('semver', {'min_version': '2.8.0'}),
 )
+
+TESTS_REQUIRES = (
+    ('pytest', {'min_version': '4.6.2'}),
+    ('pytest-cov', {'min_version': '2.5.1'}),
```

Comment:
Should not be required   
But will be needed on CI if coverage is added

Reply:
So thinking ahead! Without realizing it :)